codice/ddf
# DDF-2388 Aligned GMD Metacard Mappings and Metacard Transformer with new taxonomy and added additional mappings

#### What does this PR do?
This PR updates GMD Metacard Mappings, and adds additional query-able apiso properties to the CSW Endpoint.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@glenhein @jrnorth 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@kcwire 
#### How should this be tested?
Build and Install DDF, ingest GMD XML and verify endpoint queries and updates via postman
#### Any background context you want to provide?
#### What are the relevant tickets?
https://codice.atlassian.net/browse/DDF-2388
#### Screenshots (if appropriate)
#### Checklist:
